magento/pwa-studio
# [PWA-1033] Localize additional strings

## Description

Localize remaining client-side strings from newly delivered features, as well as from strings found using the new `jsx-no-literals` ESLint rule

## Related Issue





https://jira.corp.magento.com/browse/PWA-1033

## Acceptance 



### Verification Stakeholders

@sirugh 
@dpatil-magento 
### Specification



### Verification Steps


1. Go areas of the app where code changes occurred.
- NotFound
- Account Information Page
- App
- Category List
- Checkout Page
- CMS Block
- My Account, Reset Password
- Order History Page
- Wishlist Page, Create Wishlist
2. Verify strings are localized.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
